src/routes/projects/GeneralInfo.svelte

```diff
@@ -0,0 +1,18 @@
+<div
+    class="m-0 rounded-2xl bg-csi-neutral-50 px-8 py-8 font-inter text-csi-black shadow-lg lg:max-w-fit dark:bg-csi-neutral-900"
+>
```

Comment:
The text color currently isn't changing in dark mode. One way to address this is by adding a `dark:text-csi-white` would have solved the issue.

However, instead, I suggest using `prose` and `dark:prose-invert` (from [`tailwindcss/typography`](https://github.com/tailwindlabs/tailwindcss-typography)) which applies not only the appropriate text colors but also the default font family for the website.  

```suggestion
<div
    class="prose m-0 rounded-2xl bg-csi-neutral-50 px-8 py-8 shadow-lg dark:prose-invert lg:max-w-fit dark:bg-csi-neutral-900"
>
```